Modules/dnnGlimpse/dnnGlimpse.dnn

```diff
@@ -107,6 +107,11 @@
               <name>Glimpse.Ado.dll</name>
               <version>01.07.03</version>
             </assembly>
+            <assembly>
+              <path>bin</path>
+              <name>Glimpse.Ado.dll</name>
```

Comment:
```suggestion
              <name>Glimpse.EF6.dll</name>
```

Reply:
It needs both. :)

Reply:
Right, `Glimpse.Ado.dll` is already listed on line 107

Reply:
D'oh!  I missed that.  Yes, you're right.  :D

Reply:
FYI - I pushed this update,  I probably should have just clicked the "Commit Suggestion" button.  :)